styles/index.css

```diff
@@ -52,3 +116,83 @@
   font-family: var(--heading-sans);
   font-weight: 600;
 }
+.left-corner p {
+  margin-left: 0;
+}
+.blogs-section-container {
+  padding: 30px 88px;
+  font-family: var(--paragraph-sans);
+}
+.blogs-header-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+.blogs-header-container h1,
+p {
+  margin: 10px;
+}
+.blogs-header-container h1 {
+  padding: 0;
+}
+.blogs-carousel {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.left-button,
+.right-button {
+  width: 15px;
+  height: 15px;
+  border: none;
+  background-color: inherit;
+  cursor: pointer;
+  padding: 0;
+  margin: 0;
+}
+.carousel-buttons {
+  display: flex;
+  gap: 10px;
+  margin-bottom: 10px;
```

Comment:
move this to parent container instead. Content items should not have to worry about spacing against other container